graphql-spring-boot-test/src/main/java/com/graphql/spring/boot/test/GraphQLTestSubscription.java

```diff
@@ -205,8 +205,23 @@ public void reset() {
    * @param timeout timeout in milliseconds. Test will fail if no message received from the
    *     subscription until the timeout expires.
    * @return The received response.
+   * @see #awaitAndGetNextResponse(Duration)
+   * @deprecated since 12.0.0. Use {@link #awaitAndGetNextResponse(Duration)} instead.
    */
+  @Deprecated
   public GraphQLResponse awaitAndGetNextResponse(final int timeout) {
+    return awaitAndGetNextResponse(Duration.ofMillis(timeout));
+  }
+
+  /**
+   * Awaits and returns the next response received from the subscription. The subscription will be
+   * stopped after receiving the message (or timeout).
+   *
+   * @param timeout timeout in durations. Test will fail if no message received from the
```

Comment:
"timeout in durations" sounds a bit strange. Simply "Timeout duration" seems more appropriate